src/m365/spfx/commands/project/project-github-workflow-add.ts

```diff
@@ -155,6 +157,24 @@ class SpfxProjectGithubWorkflowAddCommand extends BaseProjectCommand {
       workflow.on.push.branches[0] = options.branchName;
     }
 
+    const version = this.getProjectVersion();
+
+    if (!version) {
+      throw 'Unable to determine the version of the current SharePoint Framework project';
```

Comment:
Maybe we should guide the user here a bit on why we couldn't find the SPFx version. We could state that the version was not found in the `.yo-rc.json` file.